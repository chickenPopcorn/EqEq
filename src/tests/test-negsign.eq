SomeContext = {
  a = { 42.333; }
}

SomeContext:find a {
  print("%.0f\n", -a);
  print("%.0f\n", - 1);
  print("%.0f\n", - a);
  print("%.0f\n", 1 - -a);
  print("%.0f\n", 1 --a);
  print("%.0f\n", a+1);
  print("%.0f\n", 1 + a);
}