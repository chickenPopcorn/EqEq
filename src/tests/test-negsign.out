-42
-1
-0.0012
0.0012
2699999999.9988
2700000042.3330
-42
43
43
43
41
43
